eventlet/zipkin/api.py

```diff
@@ -155,10 +155,11 @@ def build_span(name, trace_id, span_id, parent_id,
 
     @staticmethod
     def build_annotation(value, endpoint=None):
-        if isinstance(value, unicode):
+        if isinstance(value, str):
             value = value.encode('utf-8')
+        assert isinstance(value, bytes)
         return ttypes.Annotation(time.time() * 1000 * 1000,
-                                 str(value), endpoint)
```

Comment:
I don't think we should remove that cast.
ttypes.Annotation expect a string: https://github.com/eventlet/eventlet/blob/master/eventlet/zipkin/_thrift/zipkinCore/ttypes.py#L143 and your previous check want to force for type bytes.

Reply:
See above.

Reply:
thanks